src/content/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model.mdx

```diff
@@ -724,9 +724,9 @@ Admin and manager capabilities for Infrastructure include:
 
 ## Insights permissions
 
-Here is a summary of Admin and Add-on manager capabilities with [New Relic Insights](/docs/insights/use-insights-ui/getting-started/introduction-new-relic-insights).
+Here is a summary of Admin and Add-on manager capabilities with [New Relic Insights](/docs/insights/use-insights-ui/getting-started/introduction-new-relic-insights). 
 
-To allow a User or Restricted User to execute any of these functions in New Relic Insights, assign an Insights manager role. These functions include:
+To allow a User or Restricted User to execute any of these functions, assign an Insights manager role. These functions include:
```

Comment:
Not sure why User and Restricted User are capitalized...

Reply:
it's holdover from original user model where for some reason they capitalized them. these are the old docs which we're not touching much at this point